dotCMS/src/main/java/com/dotcms/graphql/business/CountTypeProvider.java

```diff
@@ -0,0 +1,28 @@
+package com.dotcms.graphql.business;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static graphql.Scalars.GraphQLString;
+
+import com.dotcms.graphql.util.TypeUtil;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLOutputType;
+import graphql.schema.GraphQLType;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+public enum CountTypeProvider implements GraphQLTypesProvider {
+
+    INSTANCE;
+
+    final Map<String, GraphQLOutputType> pageFields = map(
+            "totalCount", GraphQLString,
+            "fieldName", GraphQLString);
```

Comment:
I am not sure if this should be fieldName, maybe 'collectionName' could be a better name

Reply:
Waiting on discussion with team

Reply:
leaving like this